lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback-permalink.scss

```diff
@@ -0,0 +1,58 @@
+.reportback__permalink {
+  .header {
+    > .wrapper {
+      text-align: left;
+    }
+  }
+
+  .container {
+    &.-purple {
+      background: $purple;
+    }
+
+    .-white {
+      background: $white;
+    }
+
+    > .wrapper {
+      @include span(16);
```

Comment:
@weerd hmm i dunno because there is some styling that is not just specific to the card, like the header alignment on mobile, and the purple background.